packages/scripts/README.md

```diff
@@ -378,6 +378,13 @@ In the case where the plugin author wants to customize the files included in the
 
 It reuses the same logic as `npm pack` command to create an npm package tarball.
 
+This is how you create a custom root folder inside the zip file.
+- When updating a plugin, WordPress expects a folder in the root of the zip file which matches the plugin name. So be aware that this may affect the plugin update process.
```

Comment:
```suggestion
This is how you create a custom root folder inside the zip file.

-   When updating a plugin, WordPress expects a folder in the root of the zip file which matches the plugin name. So be aware that this may affect the plugin update process.
```

Nit